### PR TITLE
Auth0 and Profile Updates

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
image

pulled from gravatar
image

in the header:
image

### CODE REVIEW

File: source/SIL.AppBuilder.Portal.Frontend/src/lib/auth0.ts [hunk 1]
```diff
@@ -80,6 +86,15 @@ let lockInstance: Auth0LockStatic;
 export function getAuth0LockInstance(options = {}): Auth0LockStatic {
   if (lockInstance === undefined) {
     lockInstance = new Auth0Lock(auth0Env.clientId, auth0Env.domain, {
+      // TODO: pull language form i18next
+      language: 'en',
```
Comment (chrisvire) [inline source/SIL.AppBuilder.Portal.Frontend/src/lib/auth0.ts]: locale is available from google-auth2 provider jwt.

File: source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.ts [hunk 2]
```diff
@@ -55,6 +55,7 @@ export default {
     pictureTitle: 'Profile Picture',
     general: 'General',
     updated: 'User updated successfully!',
+    updatePicture: 'Update your picture at Gravatar.com',
```
Comment (chrisvire) [inline source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.ts]: Is it guaranteed that google-auth2 get picture from Gravatar.com?

File: source/SIL.AppBuilder.Portal.Frontend/src/lib/auth0.ts (hunk 1 above)
Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/lib/auth0.ts]: this is before we have the jwt.

File: source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.ts (hunk 2 above)
Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.ts]: this isn't related to google-auth2 --- but yeah, I guess maybe the picture is gotten from other locations. no idea atm.

**Comment by chrisvire** [comment]
May need to change "Update your picture  at Gravatar.com" or remove it in the future.  Fine for now.

**Pull request merged**